File: lab-2/src/pokemons/Garchomp.java
package pokemons;

import moves.*;
import ru.ifmo.se.pokemon.*;

public class Garchomp extends Gabite {
    public Garchomp(String name, int level) {
        super(name, level);
        setStats(108,130,95,80,85,102);
        setType(Type.DRAGON, Type.GROUND);
        setMove(new EnergyBall());
    }
}

File: lab-2-kotlin/src/pokemons/Garchomp.kt
package pokemons

import moves.EnergyBall

import ru.ifmo.se.pokemon.*

class Garchomp(name: String?, level: Int) : Gabite(name, level) {
    init {
        setStats(108.0, 130.0, 95.0, 80.0, 85.0, 102.0)
        setType(Type.DRAGON, Type.GROUND)
        setMove(EnergyBall())
    }
}